cudapoa/benchmarks/CMakeLists.txt

```diff
@@ -9,8 +9,7 @@
 #
 
 # Add benchmarks
-add_subdirectory(single-batch)
-add_subdirectory(multi-batch)
+add_subdirectory(run-batch)
```

Comment:
Instead of keeping them in a `run-batch` folder, let's move all the files up to the `benchmarks` directory directly.